tests/common/plugins/pdu_controller/controller_base.py

```diff
@@ -1,56 +1,56 @@
 """
-Base class for controlling PSUs of DUT
+Base class for controlling PDUs conecting to DUT
```

Comment:
Typo. However, suggest rewording: `Base class for controlling PDU(s) connected to DUT power supplies`

Reply:
Thanks Joe! Fixed.